templates/gp-templates-overrides/locale-projects.php

```diff
@@ -0,0 +1,306 @@
+<?php
```

Comment:
I'd suggest not to override the whole file but to add the customization directly in https://github.com/WordPress/wordpress.org/blob/trunk/wordpress.org/public_html/wp-content/plugins/wporg-gp-customizations/templates/locale-projects.php